e2e/checkpointer_test.go

```diff
@@ -24,117 +24,9 @@ const (
 )
 
 func TestCheckpointer(t *testing.T) {
-	t.Run("UnscheduleCheckpointer", testCheckpointerUnscheduleCheckpointer)
 	t.Run("UnscheduleParent", testCheckpointerUnscheduleParent)
 }
 
-// 1. Schedule a pod checkpointer on worker node.
-// 2. Schedule a test pod on worker node.
-// 3. Reboot the worker without starting the kubelet.
-// 4. Delete the checkpointer on API server.
-// 5. Reboot the masters without starting the kubelet.
-// 6. Start the worker kubelet, verify the checkpointer and the pod are still running as static pods.
-// 7. Start the master kubelets, verify the checkpointer is removed but the pod is still running.
-func testCheckpointerUnscheduleCheckpointer(t *testing.T) {
```

Comment:
can you do t.skip("skip is for now since XXX") instead of removing it completely? or we do want to remove this forever?

Reply:
:+1: for skipping